app/lib/screens/history/history.dart

```diff
@@ -23,6 +60,22 @@ class _TravelHistoryState extends State<TravelHistory> {
           style: Theme.of(context).textTheme.headline5,
         ),
       ),
+      floatingActionButton: FloatingActionButton(
+        onPressed: () async {
+          buildPdf();
+          savePdf();
+          final output = await getTemporaryDirectory();
```

Comment:
I see this warning in my IDE upon inspecting your code:
![Screen Shot 2021-01-13 at 8 00 11 PM](https://user-images.githubusercontent.com/32527219/104535187-4e9c5880-55db-11eb-9bcf-c14b08b30c35.png)

- [ ] Please fix or find out why

Reply:
@holtzmak I don't see this into my IDE again. I think once go to `pubspec.yaml` file and do `pub get` & `pub upgrade` snd restart IDE, will be able to resolve the issue.